### DISCUSSION TITLE
aaa53108-6622-4c27-a233-da0c8a7e6d67

### DISCUSSION THREAD

**Discussion by giscus** [varofla/blog_giscus#18] [General]
# aaa53108-6622-4c27-a233-da0c8a7e6d67

우피 노션 블로그에 giscus를 이용한 댓글 기능을 추가하고 댓글을 켜고 끄는 커스텀 yaml 코드 기능을 만들어보았습니다.

https://www.varofla.com/aaa53108-6622-4c27-a233-da0c8a7e6d67

**Comment by ella-yschoi** [comment]
글 잘 참고하여 적용했습니다. 감사합니다! 그런데 코드 중 comment가 모두 'commant'로 되어있는데, 따로 의미를 두신 것이 아니라면 comment가 어떨지 제안 드립니다 :)

**Reply by varofla** [reply]
으아아아악 ㅜㅜㅜ.... 도움이 되셨다니 다행이에요 함수 이름 빼고 ㅜㅜ...
본문 바로 바꿔두겠습니다 감사합니다..!
